plugins/ExamplePlugin/Updates/0.0.2.php

```diff
@@ -18,11 +18,25 @@
  */
 class Updates_0_0_2 extends Updates
 {
+
     /**
-     * Here you can define one or multiple SQL statements that should be executed during the update.
-     * @return array
+     * Return SQL to be executed in this update.
+     *
+     * SQL queries should be defined here, instead of in `doUpdate()`, since this method is used
+     * in the `core:update` command when displaying the queries an update will run. If you execute
+     * queries directly in `doUpdate()`, they won't be displayed to the user.
+     *
+     * @param Updater $updater
+     * @return array ```
+     *               array(
+     *                   'ALTER .... ' => '1234', // if the query fails, it will be ignored if the error code is 1234
+     *                   'ALTER .... ' => false,  // if an error occurs, the update will stop and fail
+     *                                            // and user will have to manually run the query
+     *               )
+     *               ```
+     * @api
```

Comment:
Small detail but I'm not sure the `@api` makes sense here since it's an example class. The API is actually in the `Updates` class right?